src/System.Net.Sockets/tests/FunctionalTests/SendReceive.cs

```diff
@@ -949,6 +949,117 @@ protected static (Socket, Socket) CreateConnectedSocketPair()
                 return (client, server);
             }
         }
+
+        [Fact]
+        [PlatformSpecific(~TestPlatforms.OSX)] // Not supported on OSX.
```

Comment:
why not? This should work on OSX if we call shutdown()

Reply:
We are calling shutdown. I think CI showed this failed on OSX, I'll check.

Reply:
ok. Let me know if you need any help with OSX. I have access to few as well as CI repro machines.

Reply:
UdpReceiveGetsCanceledByDispose fails on OSX with Shutdown call.
The only test that works on OSX with Shutdown call is TcpReceiveSendGetsCanceledByDispose, the others fail:

https://github.com/dotnet/corefx/pull/37486/commits/5ab7fce93c9f95a07ce8a1a3cdd3839ca4e6eb31 CI: https://mc.dot.net/#/user/dotnet-bot/pr~2Fdotnet~2Fcorefx~2Frefs~2Fpull~2F37486~2Fmerge/test~2Ffunctional~2Fcli~2F/20190513.18/workItem/System.Net.Sockets.Tests